test/boost/CMakeLists.txt

```diff
@@ -264,6 +264,8 @@ add_scylla_test(top_k_test
   KIND BOOST)
 add_scylla_test(transport_test
   KIND SEASTAR)
+add_scylla_test(bti_node_sink_test
+  KIND BOOST)
```

Comment:
Actual test missing

Reply:
Right, I moved the test to a separate commit and forgot to move the `configure.py`/`CMakeLists.txt` additions across commits accordingly.

Reply:
Fixed.